src/components/organisms/footer/footer.html

```diff
@@ -0,0 +1,57 @@
+<section class="footer">
```

Comment:
```suggestion
<footer class="footer">
```
いちおうfooterタグがあるのでそちらがいいかなと思います！